### PR TITLE
Tweak notification spinner behavior

### PR DESCRIPTION
**Pull request by akalin-keybase**
Show the spinner for a minimum of 300ms.

Changed the "Saved" timeout to 2.5s.

Change "Saved" text color to green2.

### CODE REVIEW

File: shared/chat/conversation/info-panel/notifications/container.js [hunk 1]
```diff
@@ -84,14 +90,14 @@ export default compose(
       },
       updateDesktop: (state, props) => (desktop: Types.NotificationsType) => {
         if (desktop === state.desktop) {
-          return {}
+          return undefined
```
Comment (akalin-keybase) [inline shared/chat/conversation/info-panel/notifications/container.js]: according to the recompose docs, returning undefined skips a re-layout

**Comment by akalin-keybase** [comment]
also cc @buoyad

**Comment by akalin-keybase** [comment]
also cc @cecileboucheron, in case you wanna try out the new behavior.

**Comment by cecileboucheron** [comment]
LGTM! Small color change though, the word "Saved" should be green2, not green (icon stays green).

**Comment by akalin-keybase** [comment]
Changed to use green2!

File: shared/chat/conversation/info-panel/notifications/container.js (hunk 1 above)
Comment (chrisnojima) [reply shared/chat/conversation/info-panel/notifications/container.js]: i think {} also works but this is fine https://github.com/acdlite/recompose/blob/master/src/packages/recompose/withStateHandlers.js#L34

Comment (chrisnojima) [reply shared/chat/conversation/info-panel/notifications/container.js]: can also just 'return' don't need to return undefined

Comment (akalin-keybase) [reply shared/chat/conversation/info-panel/notifications/container.js]: Ah, you're right, I had assume they special-cased `undefined`. I'll just revert back.

Comment (chrisnojima) [reply shared/chat/conversation/info-panel/notifications/container.js]: nah lets leave it, just make it `return`

Comment (akalin-keybase) [reply shared/chat/conversation/info-panel/notifications/container.js]: Done

**Pull request merged**